rllib/policy/torch_policy_v2.py

```diff
@@ -614,14 +627,16 @@ def learn_on_batch(self, postprocessed_batch: SampleBatch) -> Dict[str, TensorTy
         # Compute gradients (will calculate all losses and `backward()`
         # them to get the grads).
         grads, fetches = self.compute_gradients(postprocessed_batch)
+        print("fetches: ", fetches)
```

Comment:
Remember to remove this after debugging ;)

Reply:
done